bodhi/tests/server/test_alembic.py

```diff
@@ -22,14 +22,15 @@
 
 import os
 import subprocess
-import unittest
+
+from bodhi.tests.server.base import BasePyTestCase
```

Comment:
```suggestion
```

Reply:
this looks like it is lo longer used, and importing it is causing the flake8 style tests to fail in the CI.